Dockerfile.ubi

```diff
@@ -2,11 +2,8 @@
 ARG BASE_UBI_IMAGE_TAG=9.5-1742914212
 ARG PYTHON_VERSION=3.12
 
-ARG TORCH_CUDA_ARCH_LIST="7.0 7.5 8.0 8.6 8.9 9.0+PTX"
-ARG vllm_fa_cmake_gpu_arches='80-real;90-real'
-
 ## Base Layer ##################################################################
-FROM registry.access.redhat.com/ubi9/ubi-minimal:${BASE_UBI_IMAGE_TAG} as base
+FROM registry.access.redhat.com/ubi9/ubi-minimal:${BASE_UBI_IMAGE_TAG} AS base
```

Comment:
could you please try with a base image built by AIPCC: http://quay.io/aipcc/base-images/cuda
if your builder doesn't have access to that image, you can ping me on Slack and I'll send you the procedure to get access.

Reply:
also for reference, this is the dockerfile for RHAIIS: https://gitlab.com/redhat/rhel-ai/rhaiis/containers/-/blob/main/Containerfile.cuda-ubi9?ref_type=heads